modules/serverless-ossm-setup-without-kourier-mtls.adoc

```diff
@@ -0,0 +1,199 @@
+     port: 80
+     targetPort: 8081
+----
+<1> Add the name of your wildcard certificate.
+<2> The `knative-local-gateway` serves HTTP traffic. That means that traffic coming from outside of {ProductShortName}, but using an internal hostname, such as `example.default.svc.cluster.local` is not encrypted. You can set up encryption for this path by creating another wildcard certificate and an additional gateway that uses a different `protocol` spec.
++
+.Example `knative-local-gateway` using HTTPS
+[source,yaml]
+----
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: knative-local-gateway
+  namespace: knative-serving
+spec:
+  selector:
+    istio: ingressgateway
+  servers:
+    - port:
+        number: 443
+        name: https
+        protocol: HTTPS
+      hosts:
+        - "*"
+      tls:
+        mode: SIMPLE
+        credentialName: <wildcard_certs>
+----
+. Apply `Gateway` resources:
++
+[source,terminal]
+----
+$ oc apply -f <filename>
+----
+. Install Knative Serving by creating the `KnativeServing` custom resource definition (CRD), and enable Istio integration:
++
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+  namespace: knative-serving
+spec:
+  ingress:
+    istio:
+      enabled: true <1>
+  deployments: <2>
+  - name: activator
+    annotations:
+      "sidecar.istio.io/inject": "true"
+      "sidecar.istio.io/rewriteAppHTTPProbers": "true"
+  - name: autoscaler
+    annotations:
+      "sidecar.istio.io/inject": "true"
+      "sidecar.istio.io/rewriteAppHTTPProbers": "true"
+----
+<1> Enables Istio integration.
+<2> Enables sidecar injection for Knative Serving data plane pods.
+. Apply the `KnativeServing` resource:
++
+[source,terminal]
+----
+$ oc apply -f <filename>
+----
+. Create a Knative Service that has sidecar injection enabled and uses a pass-through route:
++
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: <service_name>
+  namespace: default
+  annotations:
+    serving.knative.openshift.io/enablePassthrough: "true" <1>
+spec:
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "true" <2>
+        sidecar.istio.io/rewriteAppHTTPProbers: "true"
+    spec:
+      containers:
+      - image: <image_url>
+----
+<1> Instructs Knative Serving to generate an OpenShift pass-through enabled route, so that the certificates you have generated are served through the ingress gateway directly.
+<2> Injects {ProductShortName} sidecars into the Knative service pods.
+. Apply the `Service` resource:
++
+[source,terminal]
+----
+$ oc apply -f <filename>
+----
+
+.Verification
+
+* Access your serverless application by a secure connection that is now trusted by the CA:
++
+[source,terminal]
+----
+$ curl --cacert wildcard.crt <service_url>
+----
++
+.Example command
+[source,terminal]
+----
+$ curl --cacert wildcard.crt https://hello-default.example.com
```

Comment:
Those examples should use the `root.crt` as a CA (instead of `wildcard.crt`).

Also, I think we should add here the same commands as we do on TLS cert doc: https://deploy-preview-33179--osdocs.netlify.app/openshift-enterprise/latest/serverless/service_mesh/serverless-ossm-tls-with-kourier.html#serverless-ossm-v2x-tls_serverless-ossm-tls

Reply:
```suggestion
$ curl --cacert root.crt <service_url>
----
+
.Example command
[source,terminal]
----
$ curl --cacert root.crt \
    --header "Host: hello-default.example.com" \
    --resolve "hello-default.example.com:443:<ingress_router_IP>" \
     https://hello-default.example.com
```